packages/bus/src/RetryStrategies/ExponentialBackoffRetryStrategy.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ExponentialBackoffRetryStrategy } from './ExponentialBackoffRetryStrategy';
+import { instance, mock, reset, verify, when } from 'ts-mockito';
+
+describe('ExponentialBackoffRetryStrategy', () => {
+  let retryStrategy!: ExponentialBackoffRetryStrategy;
+  const mockTest = mock<Test>();
+
+  beforeEach(() => {
+    retryStrategy = new ExponentialBackoffRetryStrategy({ maxDepth: 1 });
+  });
+
+  afterEach(() => {
+    reset(mockTest);
+  });
+
+  it('should not retry if function not throw error', async () => {
+    when(mockTest.test()).thenReturn();
+    await expect(
+      retryStrategy.acquire(() => instance(mockTest).test())
+    ).resolves.not.toThrow();
+    verify(mockTest.test()).once();
+  });
+
+  it('should not do 5 retries if error have not correct code', async () => {
```

Comment:
First of all, it won't retry 5ht times since `maxDepth` is 1. Secondly, 

```suggestion
  it('should prevent retries if error does not have a correct code', async () => {
```